### ISSUE TITLE
Add zzc_sign.py to PyPI

### ISSUE THREAD

**Issue by Maxr1998** [jixunmoe/qmweb-sign#1]
Just like the JavaScript implementation is in the npm registry, would it be possible to submit the Python implementation to a package repository, e.g., PyPI?

Very interesting write-up, by the way, and good job on reverse-engineering this algorithm!

**Comment by jixunmoe** [comment]
Probably easier to just copy and paste the file. I'm not familiar with Python package repositories.

**Comment by Maxr1998** [comment]
Fair enough. As it's just a single file that definitely works, it's just not ideal from a copyright and licensing point of view.

If you like, I can certainly help and prepare the necessary files for submission and open a PR. Then, you'd only need a PyPI account and upload the package.

**Comment by jixunmoe** [comment]
It is licensed under MIT, add a link to this repo's license and done :)
